extensions/wikia/CrosslinkModule/CrosslinkModuleController.class.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+class CrosslinkModuleController extends WikiaController {
+
+	const DEFAULT_TEMPLATE_ENGINE = WikiaResponse::TEMPLATE_ENGINE_MUSTACHE;
```

Comment:
I'd be more comfortable with
    $this->response->setTemplateEngine( WikiaResponse::TEMPLATE_ENGINE_MUSTACHE );